### PR TITLE
[format-messy-date] Add typings

### PR DESCRIPTION
**Pull request by isitreallyalive**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/format-messy-date/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// Type definitions for format-messy-date 1.1
+// Project: https://github.com/baz/foo (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
+// Definitions by: newtykins <https://github.com/me>
```
Comment (DangerBotOSS) [inline types/format-messy-date/index.d.ts:3]: - :no_entry_sign: This line should have your github username in it, not /me

Comment (newtykins) [reply types/format-messy-date/index.d.ts]: Fixed in the latest commit

File: types/format-messy-date/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+// Type definitions for format-messy-date 1.1
+// Project: https://github.com/alvacoder/format-messy-date
+// Definitions by: newtykins <https://github.com/newtykins>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface Options {
+    now?: boolean;
```
Comment (domdomegg) [inline types/format-messy-date/index.d.ts:7]:
```suggestion
    now?: Date;
```

**Review by domdomegg — CHANGES_REQUESTED** [review]
As per code comment

File: types/format-messy-date/index.d.ts (hunk 2 above)
Comment (newtykins) [reply types/format-messy-date/index.d.ts]: Fixed in the latest commit

**Comment by newtykins** [comment]
I'm confused as to what all these Mergebot Errors are about.

**Comment by domdomegg** [comment]
I am similarly incredibly confused - this looks like a potential bug in the DT tooling? If the definitions haven't been correctly merged happy for you to open another PR and ping me as a reviewer. Apologies for this.

**Pull request merged**